Configure IPA with useful inspection settings

priteau opened the pull request:
Add configuration to include extra-hardware and mellanox elements, with other useful collectors and kernel settings.

markgoddard commented on etc/kayobe/ipa.yml:
```diff
@@ -5,7 +5,7 @@
 
 # Whether to build IPA images from source.
-#ipa_build_images:
+ipa_build_images: True
```
This does change the workflow a bit - now you need to build an image, both for bifrost & overcloud ironic. I'd at least avoid putting this in wallaby.

My medium term goal is to build IPA images and put them on Ark, then just include config to pull them down in here. The build flag would go in the `ci-builder` environment.

priteau replied:
I removed `ipa_build_images` for now, it is up to the operator to enable it.

markgoddard commented:
What should we do here?

priteau commented:
> What should we do here?

Are we still planning to build IPA images and host them on Ark?

markgoddard commented:
> > What should we do here?
> 
> Are we still planning to build IPA images and host them on Ark?

That is in the plan, although not scheduled in the near future.

priteau commented:
> > > What should we do here?
> > 
> > 
> > Are we still planning to build IPA images and host them on Ark?
> 
> That is in the plan, although not scheduled in the near future.

Thanks. I need to revisit this because IPA images on recent releases will use CentOS Stream 9.

MoteHue reviewed (dismissed):
LGTM

markgoddard commented on etc/kayobe/ipa.yml:
```diff
@@ -134,6 +139,10 @@
 
 # List of additional kernel parameters for Ironic python agent.
 #ipa_kernel_options_extra:
```
Need to uncomment?

priteau replied:
D'oh!

priteau committed:
Configure IPA with useful inspection settings

Add configuration to include extra-hardware and mellanox elements, with
other useful collectors and kernel settings.

Alex-Welsh commented:
I've resolved the final conversation so this is now mergable, but it was last approved over a month ago. @priteau or @MoteHue could you quickly check it's not stale since then, and click the button?

MoteHue commented:
> I've resolved the final conversation so this is now mergable, but it was last approved over a month ago. @priteau or @MoteHue could you quickly check it's not stale since then, and click the button?

Still LGTM, let's merge :)
